### DISCUSSION TITLE
Poll bgp sessions

### DISCUSSION THREAD

**Discussion by NiteCrwlr** [peering-manager/peering-manager#612] [Q&A] [answered]
Hi

I just upgraded to v1.6.3 and want to poll bgp sessions, but nothing happens.
No log in logs/peering-manager.log and on the command line I see only this:

/opt/peering-manager# ./manage.py poll_bgp_sessions -v 3
[*] Polling BGP sessions state

Any idea to debug this?

**Comment by gmazoyer** [comment, marked as answer]
Do the routers have their `Poll BGP sessions state` field checked?

**Reply by NiteCrwlr** [reply]
Pah, that was it! Sorry, I did not notice that change. Tomatoes on my eyes...
Thank you!
